setup.py

```diff
@@ -11,7 +11,7 @@
 with open("HISTORY.rst") as history_file:
     history = history_file.read()
 
-requirements = ["Click>=6.0", "pandas", "requests"]
+requirements = ["Click>=6.0", "pandas", "requests", "numpy"]
```

Comment:
I'm not sue this is needs since the only reason for numpy appears to be an unnecessary use of np.nan currently. Either way numpy will be a requirement, as pandas depends on it, but I don't see yet how our use case will involve explicit need for numpy, which this requirement suggests.